timeseer/podman/roles/timeseer_data_service/templates/timeseer-data-service-container.quadlet.j2

```diff
@@ -0,0 +1,31 @@
+[Unit]
+Wants=network.target
+After=network.target
+
+
+[Container]
+ContainerName=timeseer-data-service
+Environment=RUST_LOG=info
+Exec=/usr/bin/timeseer-data-service
+LogDriver=journald
```

Comment:
The order of the statements here is inconsistent with the previous one. This is confusing to people editing multiple quadlet files.